managing-os/templates/fedora/upgrade-27-to-28.md

```diff
@@ -0,0 +1,423 @@
+  If you wish, you can help us test these instructions by using <code>--enablerepo=qubes-vm*current-testing</code>.
+  Once the packages have migrated to the stable repositories for both versions, this notice will be removed.
+</div>
+
+This page provides instructions for performing an in-place upgrade of an
+installed Fedora 27 [TemplateVM] to Fedora 28. If you wish to install a new,
+unmodified Fedora 28 template instead of upgrading a template that is already
+installed in your system, please see the [Fedora TemplateVM] page instead.
+
+These instructions can also be used to upgrade a Fedora 26 TemplateVM to
+Fedora 28. Simply start by cloning `fedora-26` instead of `fedora-27` in the
+instructions below.
+
+
+Important information regarding RPM Fusion repos
+------------------------------------------------
+
+If your RPM Fusion repositories are **disabled** when you upgrade a TemplateVM from Fedora 27 to 28, all RPM Fusion packages and RPM Fusion repo definitions will be removed from that TemplateVM.
+If your RPM Fusion repositories are **enabled** when upgrading, all RPM Fusion packages and repo definitions will be retained and updated as expected.
+For most users, this behavior should not cause a problem, since a TemplateVM in which the RPM Fusion repos are disabled is probably a TemplateVM in which you never wish to use them.
+However, if you wish to have the RPM Fusion repo definitions after upgrading in a TemplateVM in which they are currently disabled, you may wish to temporarily enable them prior to upgrading or manually create, copy, or download them after upgrading.
+
+
+Workaround for `python2-xcffib` upgrade error
+---------------------------------------------
+
+When attempting to upgrade from Fedora 26 or 27 to Fedora 28, you may encounter an error similar to this:
+
+    Error: Transaction check error:
+      file /usr/lib/python2.7/site-packages/xcffib-0.5.1-py2.7.egg-info/PKG-INFO from install of python2-xcffib-0.5.1-5.fc28.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+      file /usr/lib/python2.7/site-packages/xcffib/_ffi.pyc from install of python2-xcffib-0.5.1-5.fc28.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+      file /usr/lib/python2.7/site-packages/xcffib/_ffi.pyo from install of python2-xcffib-0.5.1-5.fc28.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+      file /usr/lib/python2.7/site-packages/xcffib/xinput.pyc from install of python2-xcffib-0.5.1-5.fc28.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+      file /usr/lib/python2.7/site-packages/xcffib/xinput.pyo from install of python2-xcffib-0.5.1-5.fc28.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+
+To work around this error:
+
+1. Upgrade everything besides `python2-xcffib` and `qubes-core-vm` (use `-x python2-xcffib -x qubes-core-vm`).
```

Comment:
My bad, it should be `qubes-gui-vm` not `qubes-core-vm`. And on 4.0 the package is named `qubes-gui-agent`. We can have universal version by specifying all three:
`-x python2-xcffib -x qubes-gui-vm -x qubes-gui-agent`